browser/components/MarkdownSplitEditor.js

```diff
@@ -136,35 +156,60 @@ class MarkdownSplitEditor extends React.Component {
   }
 
   render () {
-    const {config, value, storageKey, noteKey, linesHighlighted} = this.props
+    const {config, value, storageKey, noteKey, linesHighlighted, isStacking} = this.props
     const storage = findStorage(storageKey)
+
+    const editorStyle = {}
+    const previewStyle = {}
+    const sliderStyle = {}
+
     let editorFontSize = parseInt(config.editor.fontSize, 10)
     if (!(editorFontSize > 0 && editorFontSize < 101)) editorFontSize = 14
+    editorStyle.fontSize = editorFontSize
+
     let editorIndentSize = parseInt(config.editor.indentSize, 10)
-    if (!(editorFontSize > 0 && editorFontSize < 132)) editorIndentSize = 4
-    const previewStyle = {}
-    previewStyle.width = (100 - this.state.codeEditorWidthInPercent) + '%'
+    if (!(editorStyle.fontSize > 0 && editorStyle.fontSize < 132)) editorIndentSize = 4
+    editorStyle.indentSize = editorIndentSize
+
+    if (isStacking) {
```

Comment:
Nit. I think this would be more readable if you add it like following:
```js
editorStyle = isStacking ? 
  { width: '100%' , height: `${this.state.codeEditorHeightInPercent}%` } :
  { width: `${this.state.codeEditorWidthInPercent}%`, height: '100%' }

previewStyle = isStacking ? 
  { width: '100%', height: `${100 - this.state.codeEditorHeightInPercent}%` } :
  { width: `${100 - this.state.codeEditorWidthInPercent}%`, height: '100%' }

sliderStyle = isStacking ? 
 { left: 0, top: `${this.state.codeEditorHeightInPercent}%` } : 
 { left: `${this.state.codeEditorWidthInPercent}%`, top: 0 }
```

if/else would be also OK but I would avoid adding each prop line by line & use backtick strings to add the `%` to the style.

Reply:
I agree, I have changed the if statement however, this would override the previously set properties, so I have used object.assign to set the new properties. It might be beneficial to update the babel config to transpile the spread operator.